src/tests/tests/storage.js

```diff
@@ -164,6 +164,8 @@
   });
 
   QUnit.test("DNT does not cascade", (assert) => {
+    let settings_map = storage.getBadgerStorageObject('settings_map');
+    settings_map.setItem("checkForDNTPolicy", true);
```

Comment:
nope. nasty catch there, if your local version is false then this gets messed up. If I hit privacy badgers unit test, my settings would change the context of the test.